src/app/(BottomTabNavigation)/Profile/DeleteAccount/index.tsx

```diff
@@ -11,23 +15,46 @@ import { userInstance, User } from '../../../../types/types';
 
 function DeleteAccountScreen() {
   const [currSession, setCurrSession] = useState<User>(userInstance);
+
   useEffect(() => {
     getCurrentUserInfo().then(result => {
       setCurrSession(result);
     });
   }, []);
+
+  const deleteAccount = () => {
+    deleteCurrentUser(currSession.id);
+    signOutUser();
+  };
+
   return (
-    <View>
-      <Text>Are you sure you want to delete your account?</Text>
-      <TouchableOpacity
-        style={styles.button}
-        onPress={() => {
-          deleteCurrentUser(currSession.id);
-          signOutUser();
-        }}
-      >
-        <Text>YES</Text>
-      </TouchableOpacity>
+    <View style={styles.container}>
+      <View style={styles.headerContainer}>
+        <CasesHeader />
+      </View>
+      <View style={styles.textContainer}>
+        <Text style={styles.topText}>Delete account?</Text>
+        <Text style={styles.blurb}>
+          Deleting your account will also permanently delete any data assoicated
+          with it. This action cannot be undone.
+        </Text>
+      </View>
+      <View style={styles.buttonContainer}>
+        <TouchableOpacity
+          onPress={() => router.back()}
+          style={styles.cancelButton}
+        >
+          <X />
+          <View style={styles.gap} />
+          <Text style={styles.cancelText}>Cancel</Text>
+        </TouchableOpacity>
+        <View style={styles.gap} />
+        <TouchableOpacity onPress={deleteAccount} style={styles.confirmButton}>
+          <WhiteTrash />
+          <View style={styles.gap} />
```

Comment:
I'd avoid using empty divs to create spacing. Place both buttons in a div and use "space-between" property.